app/app-controllers/tabBarController.js

```diff
@@ -100,7 +100,18 @@ function(
             this.viewPlugin.setContentView(this.tabBar.tabViews[tabId]);
             this.activeTabId = tabId;
 
-            this.getActiveNavigationView().isActive = true;
+            var selectedTab = this.getActiveNavigationView();
+            selectedTab.isActive = true;
+
+            // TODO: Rethink isLoaded exposure
```

Comment:
@tony-dinh what needs to be rethunked? What don't you like about this approach? :)